src/gui/mzroll/downloadmanager.h

```diff
@@ -0,0 +1,44 @@
+#ifndef DOWNLOADMANAGER_H
+#define DOWNLOADMANAGER_H
+#include <QObject>
+#include <QUrl>
+#include <QNetworkReply>
+
+class QNetworkAccessManager;
+class QNetworkRequest;
+class QUrl;
+
+class DownloadManager : public QObject
+{
+    Q_OBJECT
+public:
+    DownloadManager();
+    void setRequest(const QString& url, void* requester, bool async=true);
+    bool err;
+//    enum class RequestType {
+//        ASYNC,
+//        SYNC
+//    };
```

Comment:
Please do not leave commented code around.

Reply:
Done Intentionally.
I want to have this enum. 
Can we leave it like this ?

Reply:
Oh, in that case let's add a `TODO` or something to notify readers of its future use.

Reply:
Sure